.circleci/devcluster/single-k8s.devcluster.yaml

```diff
@@ -35,9 +35,3 @@ stages:
           kubeconfig_path: ~/.kube/config
           determined_master_ip: $DOCKER_LOCALHOST
           determined_master_port: 8080
-          internal_task_gateway:
```

Comment:
Out of curiosity, why is this deleted? Was it a bit of cleanup discovered during testing?

Reply:
Yeaa, I think we only need to use a gateway for mutliRM to make tasks that create resources in the remote cluster (the cluster tied to the additional RM) accessible to the determined master, so single RM clusters don't need that config!